tasks/gulp/compile-assets.js

```diff
@@ -183,33 +184,51 @@ gulp.task('scss:compile', function (done) {
   done()
 })
 
+function convertFileNameToModuleName (filename) {
```

Comment:
I think a doc string with an example would be really helpful here

Reply:
I've added some more code comments to this function - is that helpful?

https://github.com/alphagov/govuk-frontend/pull/2426/files#diff-ded529645ff40394f7d2c28c16f2907dc01a2760fdd702a2aecae639838c8fc1R187

Reply:
Ah yeah that's very helpful, thanks!